src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+    private String color = getColor();
```

Comment:
The initialization of 'color' with 'getColor()' is unnecessary here since 'color' is being set in the constructor. You can remove this initialization and declare the variable without assigning a value.